README.md

```diff
@@ -14,4 +14,4 @@ Copy everything you need from your previous solution.
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_list-of-goods-hooks/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Daniil-Lilin.github.io/react_list-of-goods-hooks/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description
<img width="931" alt="image" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/64750958/d066021f-3768-44f8-a5cc-c46de7425e6f">